website/src/documentation/patterns/side-inputs.md

```diff
@@ -46,29 +54,34 @@ For instance, the following code sample uses a `Map` to create a `DoFn`. The `Ma
 {% github_sample /apache/beam/blob/master/examples/java/src/main/java/org/apache/beam/examples/snippets/Snippets.java tag:SideInputPatternSlowUpdateGlobalWindowSnip1
 %}
 ```
+```py
+No sample present.
+```
 
 
 ## Slowly updating side input using windowing
 
-You can read side input pcollection periodically into distinct windows.
-Later, when you apply side input to your main input, windows will be matched automatically 1:1.
-This way, you can guarantee side input consistency on the duration of the single window.
-
-To do this, you can utilize PeriodicSequence PTransform that will generate infinite sequence
-of elements with some real-time period:
-
-1. Use the PeriodicImpulse transform to generate windowed periodic sequence.
-
-    a. MAX_TIMESTAMP can be replaced with some closer boundary if you want to stop generating elements at some point.
+You can read side input data periodically into distinct PCollection windows.
+Later, when you apply the side input to your main input, each main input
```

Comment:
I think we should remove "Later" because this is a part of the overall workflow described in the previous sentence, rather than a secondary step.

So this should be: "When you apply the side input to your main input..."